openproblems/tasks/spatial_decomposition/methods/rctd.R

```diff
@@ -0,0 +1,46 @@
+library(spacexr)
```

Comment:
For readability, could we add at the top here:
```
# RCTD
# @param sce_sc SingleCellExperiment
# @param sce_sp SingleCellExperiment
# @return sce SingleCellExperiment
```

Reply:
I like that!